expr/helper/helper.go

```diff
@@ -183,18 +183,23 @@ func SummarizeValues(f string, values []float64) (float64, bool) {
 		if len(values) > 0 {
 			rv = values[len(values)-1]
 		}
+	case "count":
+		rv = float64(len(values))
 
 	default:
-		f = strings.Split(f, "p")[1]
-		percent, err := strconv.ParseFloat(f, 64)
-		if err == nil {
-			return Percentile(values, percent, true)
+		if strings.HasPrefix(f, "p") {
```

Comment:
Do we really need the percentile summarization type that is not in the [reference version](https://graphite.readthedocs.io/en/latest/functions.html?highlight=summarize#graphite.render.functions.summarize)? Even if we do, it makes sense to add the `median` that is present in reference, which we can get for free.

Reply:
I don't think they are very useful, but since we have them they might be used :(.
Added `median`